dftool

```diff
@@ -1110,25 +1125,25 @@ ARGV_FNS = {
     "veSetAllocation" : do_veSetAllocation,
     "checkpoint_feedist" : do_checkpoint_feedist,
     "initdevwallets": do_initdevwallets,
-    "manyrandom" : do_manyrandom,
-    "newacct" : do_newacct,
-    "newtoken" : do_newtoken,
-    "newVeOcean" : do_newVeOcean,
-    "newVeAllocate" : do_newVeAllocate,
-    "newdfrewards" : do_newdfrewards,
-    "newdfstrategy" : do_newdfstrategy,
+    "manyrandom": do_manyrandom,
+    "newacct": do_newacct,
+    "newtoken": do_newtoken,
+    "newVeOcean": do_newVeOcean,
+    "newVeAllocate": do_newVeAllocate,
+    "newdfrewards": do_newdfrewards,
+    "newdfstrategy": do_newdfstrategy,
     # read actions
-    "acctinfo" : do_acctinfo,
-    "chaininfo" : do_chaininfo,
-    "getrate" : do_getrate,
+    "acctinfo": do_acctinfo,
+    "chaininfo": do_chaininfo,
+    "getrate": do_getrate,
 }
 
 
 # ========================================================================
 # main
 @enforce_types
 def do_main():
-    if len(sys.argv) == 1 :
+    if len(sys.argv) == 1:
```

Comment:
Similar to above: Please revert this, and any small refactor changes. Why: outside the scope of this PR, and makes reviewing this PR harder. If you want to get them in, do a separate PR just for small refactorings, merge it to main, then merge main into here.